src/main/scala/com/microsoft/partnercatalyst/fortis/spark/Constants.scala

```diff
@@ -15,9 +15,8 @@ object Constants {
     val FeatureServiceUrlBase = "FORTIS_FEATURE_SERVICE_HOST"
     val BlobUrlBase = "FORTIS_CENTRAL_ASSETS_HOST"
     val CassandraHost = "FORTIS_CASSANDRA_HOST"
-    val ManagementBusNamespace = "FORTIS_SERVICEBUS_NAMESPACE"
-    val ManagementBusConfigQueueName = "FORTIS_SERVICEBUS_CONFIG_QUEUE"
-    val ManagementBusPolicyName = "FORTIS_SERVICEBUS_POLICY_NAME"
-    val ManagementBusPolicyKey = "FORTIS_SERVICEBUS_POLICY_KEY"
+    val ManagementBusConnectionString = "FORTIS_SB_CONN_STR"
+    val ManagementBusConfigQueueName = "FORTIS_SB_CONFIG_QUEUE"
+    val ManagementBusCommandQueueName = "FORTIS_SB_COMMAND_QUEUE"
```

Comment:
What should I be setting `ManagementBusConfigQueueName ` and `ManagementBusCommandQueueName ` to in the deployment?

Currently the deployment for SB goes as follows:
```
readonly sb_queue_site="siteMutations"
readonly sb_queue_streams="streamMutations"
```
Then
```
--from-literal=SERVICE_BUS_QUEUE_SITE="${sb_queue_site}" \
--from-literal=SERVICE_BUS_QUEUE_STREAM="${sb_queue_streams}" \
```